scriptmodules/emulators/advmame.sh

```diff
@@ -152,6 +153,7 @@ function configure_advmame() {
 
         default=0
         [[ "$version" == "0.94.0" ]] && default=1
+        addSystem $default "$md_id-$version" "arcade" "$md_inst/$version/bin/advmame %BASENAME%"
         addSystem $default "$md_id-$version" "mame-advmame arcade mame" "$md_inst/$version/bin/advmame %BASENAME%"
```

Comment:
The three keywords are assigned like this:

mame-advmame - system (used for romdir and <name> in es_systems.cfg)
arcade - platform (used in es_systems.cfg as <platform>)
mame - theme (used in es_systems.cfg as <theme)

In your arcade case, assuming you are going to make a new arcade theme part, the single parameter is fine.